src/App/VPSScan/Types.hs

```diff
@@ -17,8 +17,8 @@ import Text.URI (URI)
 import Network.HTTP.Req
 import Data.Text.Prettyprint.Doc (viaShow)
 
-data ScotlandYardOpts = ScotlandYardOpts
-  {scotlandYardUrl :: URI}
+data CoreServerOpts = CoreServerOpts
+  {fossaUrl :: URI, fossaApiKey :: Text}
```

Comment:
[nit] we should make the naming here consistent, either:
1. renaming `CoreServerOpts` to `FossaServerOpts`
or
2. `fossaUrl` -> `coreUrl`; `fossaApiKey` -> `coreApiKey`